### ISSUE TITLE
Hint used for informing user to expand/collapse a button is announced twice.

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4572]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:**3.3.2 Labels or Instructions

**Test Environment:**
- Test URL: https://clarity.design/documentation/stack-view
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto one of the collapsed stack view sections (except for static stack-view) and observe the spoken feedback of screen reader.

**Expected result:**
Appropriate announcement made should be made for the collapsed button i.e. Label, control type, state and any other relevant information should be communicated when any one of the collapsed stack view buttons are focused.

**Actual result:**
When the button is in collapsed state screen reader announces the hint word twice i.e. &quot;Expand Expand&quot;.
When the button is in expanded state screen reader announces the hint &quot;word twice Collapse Collapse&quot;.

**Note:** Label, control type and current state of the buttons are announced appropriately.

**Possible root cause:**
Same title attribute and aria-labelledby attribute is used for the same element (i.e. the stackview expand/collapse graphic element).

Proposed solution:
By standards, hint word is not required in this scenario as that is taken care by the state itself. i.e. when the user hears that the button is collapsed/expanded they know that pressing that button will expand or collapse that button. This approach also keeps the announcement less verbose. Also, the graphics used is purely for visual indication, so hiding the element from the assistive technology could be the right way to do it.
